<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WineYuanShenFpsCrackerClass</class>
 <widget class="QMainWindow" name="WineYuanShenFpsCrackerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WineYuanShenFpsCracker</string>
  </property>
  <property name="windowIcon">
   <iconset resource="WineYuanShenFpsCracker.qrc">
    <normaloff>:/icons/256x256.ico</normaloff>:/icons/256x256.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="pidBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>421</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>31</height>
     </size>
    </property>
    <property name="placeholderText">
     <string>Please input a PID</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="hideCheck">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>118</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>118</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string>HIde after Crack</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopBtn">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>150</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>75</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>75</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="fpsBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>421</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>421</width>
      <height>31</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Please ipnut a FPS</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pidBox</tabstop>
  <tabstop>fpsBox</tabstop>
  <tabstop>hideCheck</tabstop>
  <tabstop>startBtn</tabstop>
  <tabstop>stopBtn</tabstop>
 </tabstops>
 <resources>
  <include location="WineYuanShenFpsCracker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startBtn</sender>
   <signal>clicked()</signal>
   <receiver>WineYuanShenFpsCrackerClass</receiver>
   <slot>StartBtnHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>98</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>WineYuanShenFpsCrackerClass</receiver>
   <slot>StopBtnHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>438</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>441</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>StartBtnHandler()</slot>
  <slot>StopBtnHandler()</slot>
 </slots>
</ui>
